meta-mender-imx6ul/scripts/setup-mender-environment.sh

```diff
@@ -0,0 +1,308 @@
+End User License Agreement (EULA). To have the right to use these binaries in
+your images, you need to read and accept the following...
+
+EOF
+
+    sleep 3
+
+    more -d $CWD/sources/meta-freescale/EULA
+    echo
+    REPLY=
+    while [ -z "$REPLY" ]; do
+        echo -n "Do you accept the EULA you just read? (y/n) "
+        read REPLY
+        case "$REPLY" in
+        y | Y)
+            echo "EULA has been accepted."
+            echo "ACCEPT_FSL_EULA = \"1\"" >>conf/local.conf
+            EULA=Accepted
+            ;;
+        *)
+            echo "EULA has not been accepted."
+
+            # deleting build directory
+            cd $CWD && rm -rf $CWD/$#
+            clean_up && return 1
+            ;;
+        esac
+    done
+fi
+
+cat <<EOF
+
+Welcome to Freescale Community BSP
+
+The Yocto Project has extensive documentation about OE including a
+reference manual which can be found at:
+    http://yoctoproject.org/documentation
+
+For more information about OpenEmbedded see their website:
+    http://www.openembedded.org/
+
+You can now run 'bitbake <target>'
+
+Common targets are:
+    core-image-minimal
+    meta-toolchain
+    meta-toolchain-sdk
+    adt-installer
+    meta-ide-support
+    core-image-base
+    core-image-full-cmdline
+    fsl-image-gui
+
+EOF
+
+if [ -n "$generated_config" ]; then
+
+    # Trimming unwanted mender modules
+    rm -rf $CWD/sources/meta-mender/meta-mender-commercial $CWD/sources/meta-mender/meta-mender-demo $CWD/sources/meta-mender/meta-mender-qemu $CWD/sources/meta-mender/meta-mender-raspberrypi*
+
+    # Trimming unwanted meta-mender-community layers
+    rm -rf $CWD/sources/meta-mender-community/a* $CWD/sources/meta-mender-community/b* $CWD/sources/meta-mender-community/c* $CWD/sources/meta-mender-community/n* $CWD/sources/meta-mender-community/o* $CWD/sources/meta-mender-community/q* $CWD/sources/meta-mender-community/r* $CWD/sources/meta-mender-community/s* $CWD/sources/meta-mender-community/t* $CWD/sources/meta-mender-community/u* $CWD/sources/meta-mender-community/v* $CWD/sources/meta-mender-community/intel
+
+    # Appending build specific layers to bblayers.conf
+    cat $CWD/sources/meta-mender-community/meta-mender-imx6ul/templates/bblayers.conf.append >>conf/bblayers.conf
+
+    # Disabling mx6 firmware
+    sed -i '/MACHINE_FIRMWARE_append_mx6 =/c\# MACHINE_FIRMWARE_append_mx6 = " linux-firmware-imx-sdma-imx6q"' $CWD/sources/meta-freescale/conf/machine/include/imx-base.inc
+
+    # Updating machine configuration
+    cat $CWD/sources/meta-mender-community/meta-mender-imx6ul/templates/imx6ulevk.conf.new >$CWD/sources/meta-freescale/conf/machine/imx6ulevk.conf
+    cat <<EOF
+Your build environment has been configured with:
+
+    MACHINE=$MACHINE
+    SDKMACHINE=$SDKMACHINE
+    DISTRO=$DISTRO
+    EULA=$EULA
+EOF
+else
+    echo "Your configuration files at $1 have not been touched."
+fi
+
+echo "Now you can run 'bitbake core-image-base'"
+
+clean_up
```

Comment:
I assume that this one is largely based on `fsl-setup-release.sh` provided by the NXP BSP. To avoid making a copy of it you can do something like we did for the iMX8 board [here](https://github.com/mendersoftware/meta-mender-community/blob/5adaa8c5cbe332600171589476038ea014fb9268/meta-mender-imx/scripts/fsl-setup-mender.sh)